linera-storage-service/src/child.rs

```diff
@@ -37,9 +37,11 @@ impl StorageServiceSpanner {
         command
     }
 
-    async fn wait_deadness(&self) -> Result<()> {
+    /// Wait for the absence of the endpoint. If a child is eliminated
+    /// then it might takes time to wait for its absence.
+    async fn wait_absence(&self) -> Result<()> {
         for i in 1..10 {
-            let test = storage_service_check_deadness(&self.endpoint).await?;
+            let test = storage_service_check_absence(&self.endpoint).await?;
             if test {
```

Comment:
In general, the variable name `test` doesn't convey much information. I'd either rename this to `is_absent` or inline it:
```suggestion
            let test = storage_service_check_absence(&self.endpoint).await?;
            if test {
```

```suggestion
            if storage_service_check_absence(&self.endpoint).await? {
```

Reply:
Ok.